.github/workflows/docker.yml

```
@@ -1,4 +1,4 @@
-name: Build and Push Docker Image
+name: Build and Push Docker Image 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)